live-site/components/mentors-section/mentor-popup/MentorPopup.styles.ts

```diff
@@ -24,22 +25,51 @@ const StyledContainer = styled.div`
   display: flex;
   flex-direction: row;
   gap: 3em;
+  @media ${max.tabletLg} {
+    display: block;
+    width: 16em;
+    margin-top: 5em;
+    padding: 2.5em; 
+  }
+  @media ${max.tabletXs} {
+    margin-top: 5em;
```

Comment:
remove this entire media query since margin-top: 5em; is already in the above media query